db/migrate/20240716194544_add_organization_to_form_answers.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizationToFormAnswers < ActiveRecord::Migration[7.1]
+  def change
+    add_reference :form_answers, :organization, null: false, foreign_key: true
```

Comment:
Shouldn't we have a reference to FormSubmission here? Given the belongs_to relationship

Reply:
I believe there is a reference to `FormSubmission` already and this migration file only adds the `Organization` reference since we added the `acts_as_tenant(:organization)`

```ruby
create_table "form_answers", force: :cascade do |t|
    t.json "value", null: false
    t.json "question_snapshot", null: false
    t.bigint "question_id", null: false
    t.bigint "user_id", null: false
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.bigint "form_submission_id", null: false
    t.bigint "organization_id", null: false
    t.index ["form_submission_id"], name: "index_form_answers_on_form_submission_id"
    t.index ["organization_id"], name: "index_form_answers_on_organization_id"
    t.index ["question_id"], name: "index_form_answers_on_question_id"
    t.index ["user_id"], name: "index_form_answers_on_user_id"
    ```

Reply:
I also just noticed the question_id and question_snapshot reference. It would make sense to remove those two, right?

Reply:
We are actually keeping question snapshot because we want to store the question with every answer so we have a record of what the question was. Let’s keep question Id for now but remove the null false constraint so a FormAnswer can have a parent question, but doesn’t have to.

Reply:
We can fully remove user id and it’s index